contracts/Migrator.sol

```diff
@@ -28,10 +33,21 @@ contract Migrator is IMigrator {
     }
 
     function migrate(address owner_, uint256 amount_) public override {
-        require(amount_ != uint256(0),                                              "M:M:ZERO_AMOUNT");
+        require(active,                "M:M:INACTIVE");
+        require(amount_ != uint256(0), "M:M:ZERO_AMOUNT");
 
         require(ERC20Helper.transferFrom(oldToken, owner_, address(this), amount_),   "M:M:TRANSFER_FROM_FAILED");
         require(ERC20Helper.transfer(newToken, owner_, amount_ * TOKEN_SPLIT_SCALAR), "M:M:TRANSFER_FAILED");
     }
 
+    function setActive(bool active_) external {
```

Comment:
Could make this a one-time action so once migration is set to active it can not be reverted.

Reply:
Thoughts @0xfarhaan ?

Reply:
Let's keep the option to pause.